pkg/chunk/chunk_store_test.go

```diff
@@ -22,23 +22,28 @@ import (
 	"github.com/weaveworks/common/user"
 )
 
+type schemaFactory func(cfg SchemaConfig) Schema
+type storeFactory func(StoreConfig, Schema, StorageClient) (Store, error)
+
 var schemas = []struct {
 	name              string
-	fn                func(cfg SchemaConfig) Schema
+	schemaFn          schemaFactory
+	storeFn           storeFactory
```

Comment:
I was expecting this member to come in as part of the refactor to multiple stores.